mongo/mongologger/log_level.go

```diff
@@ -0,0 +1,75 @@
+// Copyright (C) MongoDB, Inc. 2017-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+package mongologger
+
+import (
+	"errors"
+	"strings"
+)
+
+// Level indicates the user's preference on logging.
```

Comment:
I don't think it does. There is a `ClientOptions` field that indicates the user's preference for logging level, but the actual `Level` type just indicates the priority of the log message.